configure.ac

```diff
@@ -0,0 +1,72 @@
+dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
+AC_PREREQ([2.60])
+define(_CLIENT_VERSION_MAJOR, 0)
+define(_CLIENT_VERSION_MINOR, 93)
+define(_CLIENT_VERSION_BUGFIX, 0)
```

Comment:
I think the only effect it has is on the output of `./configure --help`. I agree though that a way to unify everything should be found.